### PR TITLE
Updating banner text to k-12

### PR DESCRIPTION
**Pull request by mekarpeles**
Currently on prod as a hot-deploy, in response to good feedback from @jessamynwest

Updates Sponsorship banner to k-12 student library.

### Testing & Evidence

On prod.

### CODE REVIEW

**Comment by cclauss** [comment]
The terminology `K-12` is very US-centric.

File: openlibrary/templates/site/body.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
       </div>
       $# Paste next line into announcement variable to show the blue banner
       $# <strong>Title:</strong> Description &nbsp; <a href="#" data-ol-link-track="AnnouncementsBar|BlogWidget" class="btn primary">Learn More</a>
-      $ announcement = _("<strong>Together</strong>, let's build an Open Library for the World.") + ' <a class="cta-btn cta-btn--available" style="display: inline;" href="/sponsorship">Sponsor a Book</a>'
+      $ announcement = _("Parents, teachers, and students: Visit our new") + ' <a class="cta-btn cta-btn--available" style="display: inline;" href="/k-12">K-12 Student Library</a>'
```
Comment (cdrini) [inline openlibrary/templates/site/body.html:27]: Let's i18n the whole sentence for better flow. @cclauss  I agree K-12 is a little US-centric, but we can make new collections for other nation's standards.

```suggestion
      $ announcement = _("Parents, teachers, and students: Visit our new <a %s>K-12 Student Library</a>", 'class="cta-btn cta-btn--available" style="display: inline;" href="/k-12"')
```

**Comment by cdrini** [comment]
@cclauss I agree K-12 is a little US-centric, but we can make new collections for other nation's standards.

**Comment by cdrini** [comment]
Tested works.

**Pull request merged**